search-sentiments-map/routes/search.js

```diff
@@ -65,15 +67,21 @@ router.get('/:topic/:countries', (req, res) => {
 });
 
 /** 
- * Returns a JSON-formatted array of search results for all countries retrieved
- * from the Datastore.
+ * Returns a JSON-formatted array of search results from specified time range
+ * for all countries retrieved from the Datastore.
  * @param {string} topic Search topic to get data for.
+ * @param {number} timeRange The interval value for the time range.
```

Comment:
Add more detail on the time range. I think it's only a number not really an interval?